src/pages/mining.md

```diff
@@ -68,6 +68,9 @@ miner = true
 chain = "bitcoin"
 mode = "krypton"
 peer_host = "bitcoind.blockstack.org"
+#process_exit_at_block_height = 5340
+#burnchain_op_tx_fee = 5500
+#commit_anchor_block_within = 10000
```

Comment:
@friedger I wonder if it would help to add a table listing the parameters and their meaning?

Reply:
@CharlieC3 do I understand correctly that your config PR would describe these props?

Reply:
@agraebe Yup, these properties are listed in my [PR](https://github.com/blockstack/docs/pull/871).
[See here for the preview page.](https://blockstack-docs-fkm0qgrlf.vercel.app/references/node-configuration#process_exit_at_block_height-optional)

@friedger Would appreciate any feedback on my PR on config descriptions and examples, in case what I've written isn't accurate.

Reply:
thanks @CharlieC3! It would be great if you could edit the `mining.md` file to link to your references.

@friedger I think we can merge this in now